### PR TITLE
Update CHANGES for release 14.0.0

### PR DESCRIPTION
**Pull request by jeremy-lunarg**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,35 @@
...
+* Fix SPV_KHR_cooperative_matrix enumerants
+* Fix nullptr crash
+* Fix GL_ARB_shader_storage_buffer_object version
+* Fix interpolant ES error
+* Generate DebugValue for constant arguments
+* Overflow/underflow out-of-range floats to infinity/0.0 respectively
+* Support SV_ViewID keywords for HLSL
+* Implement relaxed rule for opaque struct members
+* Add BUILD_WERROR cmake option
+* Remove various warning suppression pragmas
```
Comment (arcady-lunarg) [inline CHANGES.md:25]: This doesn't need to be in the changelog, it's not a user-facing change.

**Review by arcady-lunarg — CHANGES_REQUESTED** [review]
One line that can be removed, but otherwise I think this is fine.

**Review by ncesario-lunarg — COMMENTED** [review]
LGTM after Arcady's comment is addressed.

File: CHANGES.md (hunk 1 above)
Comment (jeremy-lunarg) [reply CHANGES.md]: Removed. Thanks!

**Review by ncesario-lunarg — APPROVED** [review]

**Review by arcady-lunarg — APPROVED** [review]

**Pull request merged**